src/language/en.tsx

```diff
@@ -81,7 +81,7 @@ const en = {
         ChooseEntryIconView: 'Choose icon',
         AddEntryView: 'Add activity',
         Information: 'Information',
-        BehaviouralActivation: 'Behaviour Activation',
+        BehaviouralActivation: 'Behavioral Activation',
```

Comment:
The name would also have to be changed in `app.json` . The repo should probably also be renamed.